test/unit/integrations/sage_pay_form/sage_pay_form_helper_test.rb

```diff
@@ -24,11 +24,7 @@ def test_basic_helper_fields
     assert_equal 5, @helper.fields.size
     assert_field 'Vendor', 'cody@example.com'
     assert_field 'Amount', '5.00'
-    assert_field 'VendorTxCode', 'order-500' + @helper.identifier
-  end
-
-  def test_identifier_should_only_contain_digits
```

Comment:
Oh true, I had pulled it out originally because I was going to include the `;` in the identifier string, but I didn't end up doing that and I didn't fix the test to match. 10 seconds.